ci/install_openebs.sh

```diff
@@ -43,8 +43,9 @@ kubectl get pvc,pv
 
 kubectl get pods --all-namespaces
 
+JIVACTRL=$(kubectl get deploy -l openebs/controller=jiva-controller --no-headers | awk {'print $1'})
```

Comment:
can you change the label to "openebs.io/controller=jiva-controller". Similarly with the replica.